src/main/java/com/google/sps/models/PCAPAnalysis.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.models;
+
+public class PCAPAnalysis {
+
+  private String id;
+  private String[] ips;
+
+  public PCAPAnalysis(String id, String[] ips) {
+    this.id = id;
+    this.ips = ips;
+  }
+
+  public String getId() {
+    return id;
+  }
+
+  public void setId(String id) {
```

Comment:
you should not have a setter for ID, as it should remain unchanged after object creation.

Reply:
Fixed